aws-android-sdk-core/src/main/java/com/amazonaws/auth/BasicAWSCredentials.java

```diff
@@ -18,6 +18,10 @@
 /**
  * Basic implementation of the AWSCredentials interface that allows callers to
  * pass in the AWS access key and secret access in the constructor.
+ *
+ * It is not recommended to use this class. Instead, use short term credentials
+ * from Cognito with the AWSMobileClient.
```

Comment:
It is not recommended to use long-term credentials through this class. Instead, use short-term credentials
 vended from Amazon Cognito through {@link com.amazonaws.mobile.client.AWSMobileClient}.

Reply:
link doesn't work because core SDK is not a dependent on auth-core SDK